datasets/stdpopsim_kern.yml

```diff
@@ -0,0 +1,39 @@
+Name: stdpopsim species resources
+Description: Contains all resources (genome specifications, recombination maps, etc.) required for
+    species specific simulation with the stdpopsim package. These resources are originally from a
+	variety of other consortium and published work but are consolidated here for ease of access and
+	use. If you are interested in adding a new species to the stdpopsim resource please raise an
+	issue on the stdpopsim GitHub page to have the necessary files added here.
+Documentation: https://stdpopsim.readthedocs.io/en/latest/catalog.html
+Contact: https://github.com/popsim-consortium/stdpopsim/issues
+ManagedBy: Andrew Kern & Jerome Kelleher
+UpdateFrequency: Data will be added as new species, genome assemblies, and genetic map data for
+    already included species become available.
+Tags:

```

Comment:
Please submit a PR to add these new tags to https://github.com/awslabs/open-data-registry/blob/master/tags.yaml

Reply:
Is it okay if I just include it in this PR?

Reply:
Yup!